alerts.go

```diff
@@ -36,9 +36,22 @@ type NotificationEmail struct {
 	Recipient *[]string `json:"recipient,omitempty"`
 }
 
+// NotificationThirdParty - Alert Rule Notification ThirdParty structure
+type NotificationThirdParty struct {
+	IntegrationId   *string `json:"integrationId,omitempty"`
```

Comment:
```suggestion
	IntegrationID   *string `json:"integrationId,omitempty"`
```

Ahh nice catch by the linter, this should be `IntegrationID` :)